emcmake.py

```diff
@@ -48,8 +70,15 @@ def has_substr(args, substr):
     elif utils.which('ninja'):
       args += ['-G', 'Ninja']
     else:
-      print('emcmake: no compatible cmake generator found; Please install ninja or mingw32-make, or specify a generator explicitly using -G', file=sys.stderr)
-      return 1
+      error('emcmake: no compatible cmake generator found; Please install ninja or mingw32-make, or specify a generator explicitly using -G')
```

Comment:
Maybe avoid this change since its unrelated?

Reply:
Sure, the change doesn't affect any behavior: just `error` is a way to print something to the console with a non-zero error code. But I can undo that change if desired.